src/upnp_common.h

```diff
@@ -0,0 +1,149 @@
+/*MT*
+    
+    MediaTomb - http://www.mediatomb.cc/
+    
+    upnp_common.h - this file is part of MediaTomb.
```

Comment:
This header should be a Gerbera one as its a new file

Reply:
done